app/assets/javascripts/controllers/analysis_controller.js

```diff
@@ -105,8 +105,10 @@
     removeFields: function() {
       $(document).on('click', '.-js-remove-fields', function(event) {
         event && event.preventDefault();
+
         $(this).prev('input[type=hidden]').val('1');
-        $(this).parents('.-js-input-wrapper').hide();
+        //I change this from hide to remove because that wrapper should be display flex and hide give it a display block. Please, review that this change doesn't crash anything.
+        $(this).parents('.-js-input-wrapper').remove();
```

Comment:
This will work OK for the analysis create form, but will probably break the analysis sidebar functionality of comparing current to future practices. By removing the elements completely we can't tell the server to ignore them, it will assume that they weren't part of the changes. Is there another way to do this? At least for the sidebar.